client/webserver/site/src/html/markets.tmpl

```diff
@@ -36,280 +36,287 @@
   </div>
 
   {{- /* RIGHT COLUMN */ -}}
-  <div class="d-flex flex-column flex-grow-1">
+  <div id="rightSide" class="d-flex flex-column flex-grow-1">
 
     {{- /* CHART */ -}}
+
     <div id="marketChart" class="w-100">
       <div id="marketLoader" class="fill-abs flex-center">
         <div class="ico-spinner spinner"></div>
 
       </div>
+      <div id="chartResizer"></div>
     </div>
 
     {{- /* ORDERS */ -}}
-    <div id="orders" class="overflow-auto w-100 d-flex flex-grow-1 bg1">
-
-      <div class="col-14 d-flex justify-content-around align-items-stretch p-0">
-
-        {{- /* BUY ORDER LIST */ -}}
-        <div class="marketorders col-7 p-0 bg1">
-          <div class="market-header text-center fs14 py-1">Buy Orders</div>
-          <table class="ordertable bg0">
-            {{block "thead" .}}
-            <thead>
-              <tr>
-                <th class="text-left pl-2">Quantity</th>
-                <th class="text-right pr-2">Rate</th>
-                <th class="text-right pr-2">Epoch</th>
-              </tr>
-            </thead>
-            {{end}}
-            <tbody id="buyRows">
-              {{- /* This row is used by the app as a template. */ -}}
-              <tr id="rowTemplate">
-                <td class="text-left pl-2" data-type="qty">-</td>
-                <td class="text-right pr-2" data-type="rate">-</td>
-                <td class="text-right pr-3" data-type="epoch"></td>
-              </tr>
-            </tbody>
-          </table>
-        </div>
-
-        {{- /* ORDER PANEL */ -}}
-        <div class="col-10 p-0 brdrleft brdrright">
-          <div class="market-header fs14 d-flex justify-content-start pl-4 align-items-stretch">
-            <button id="limitBttn" type="button" class="flex-center px-3 py-0 markettab brdrleft selected">Limit Order</button>
-            <button id="marketBttn" type="button" class="flex-center px-3 py-0 markettab brdrleft brdrright">Market Order</button>
+    <div id="orders" class="w-100 d-flex flex-grow-1 bg1">
+
+      <div class="col-14 p-0">
+        <div class="fill-abs d-flex justify-content-around align-items-stretch">
+
+          {{- /* BUY ORDER LIST */ -}}
+          <div class="marketorders col-7 p-0 bg1">
+            <div class="market-header text-center fs14 py-1">Buy Orders</div>
+            <table class="ordertable bg0">
+              {{block "thead" .}}
+              <thead>
+                <tr>
+                  <th class="text-left pl-2">Quantity</th>
+                  <th class="text-right pr-2">Rate</th>
+                  <th class="text-right pr-2">Epoch</th>
+                </tr>
+              </thead>
+              {{end}}
```

Comment:
I'm no template guru, but I can't understand what this block does.

Reply:
The block is defined here for reuse on the sells table. Look for `{{template "thead"}}` below.

Reply:
Ahhhh, I didn't think about a block being reusable.  For reason I though it was a one-time define and execute kind of thing.